service/src/test/java/org/sunbird/learner/actors/skill/UserSkillManagementActorTest.java

```diff
@@ -180,7 +183,8 @@ public void testAddSkillEndorsementSuccessWithInvalidEndorsedUserId() {
 
     mockCasandraRequestForGetUser(true);
     subject.tell(
-        createSkillEndorsementRequest(USER_ID, ENDORSED_USER_ID, ENDORSE_SKILL_NAME), probe.getRef());
+        createSkillEndorsementRequest(USER_ID, ENDORSED_USER_ID, ENDORSE_SKILL_NAME),
```

Comment:
This test-case is failure -> wrongly named to success scenario. Finally we are expecting a client_error